dbclient/metrics-jdbc/src/main/java/module-info.java

```diff
@@ -14,22 +14,24 @@
  * limitations under the License.
  */
 
-import io.helidon.dbclient.jdbc.spi.HikariCpExtensionProvider;
+import io.helidon.dbclient.jdbc.spi.JdbcCpExtensionProvider;
 import io.helidon.dbclient.metrics.jdbc.JdbcMetricsExtensionProvider;
 
 /**
  * Metrics support for Helidon Database Client JDBC.
  */
 module io.helidon.dbclient.metrics.jdbc {
-    requires io.helidon.dbclient;
-    requires io.helidon.dbclient.jdbc;
-    requires io.helidon.metrics;
-    requires io.helidon.dbclient.metrics;
-    requires com.zaxxer.hikari;
+
+    requires transitive io.helidon.dbclient;
+    requires transitive io.helidon.dbclient.jdbc;
+    requires transitive io.helidon.dbclient.metrics;
+    requires transitive io.helidon.metrics;
+
```

Comment:
This seems wrong. Why would we have a general `jdbc` metrics depending on Dropwizard metrics? This is Hikari specific and has no place here.

Reply:
This module is Hikari specific so renamed it to metrics-hikari.